verticapy/tests_new/conftest.py

```diff
@@ -448,3 +448,13 @@ def titanic_vd_fun(schema_loader):
     titanic = load_titanic(schema_loader, "titanic")
     yield titanic
     drop(name=f"{schema_loader}.titanic")
+
+
+@pytest.fixture(scope="module")
+def market_vd(schema_loader):
```

Comment:
are your test dataset agnostic or will it only work with the market dataset? If this dataset changes, will it affect the results of the tests?

Reply:
I have tried to make it dataset agnostic, however, wherever its possible. 
some of the test needs columns with specific datatype, in that case column names are hard-coded.
Few functions are not available in standard python library. so expected outcome are hard-coded. example - approx_unique and other approx function.